### PR TITLE
PSSA 1.21.0 docs changes from PSSA repo

### PR DESCRIPTION
**Pull request by bergmeister**
# PR Summary

Copied over from PSSA repo after the most rcent PR 1835 there were changes from this repo were synced back. Please note that this version isn't released yet so you might want to wait with merging or publishing until then.

Related PRs
- https://github.com/PowerShell/PSScriptAnalyzer/pull/1828

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by bergmeister** [comment]
Just saw the message around relative links. Will inspect the diff and revert those changes. Putting PR back into draft.

File: reference/ps-modules/PSScriptAnalyzer/Invoke-ScriptAnalyzer.md [hunk 1]
```diff
@@ -514,7 +514,7 @@ following keys:
 
 The keys and values in the profile are interpreted as if they were standard parameters and values of
 `Invoke-ScriptAnalyzer`, similar to splatting. For more information, see
-[about_Splatting](/powershell/module/microsoft.powershell.com/about/about_splatting).
+[about_Splatting](/powershell/module/microsoft.powershell.core/about/about_splatting).
```
Comment (bergmeister) [inline reference/ps-modules/PSScriptAnalyzer/Invoke-ScriptAnalyzer.md:517]: PSSA docs use `core` in the URL, which one should we use? https://github.com/PowerShell/PSScriptAnalyzer/blob/6790ebba18d6732fd6edf008c48e566adb508641/docs/Cmdlets/Invoke-ScriptAnalyzer.md?plain=1#L517
It seems this was maybe changed to `com` when PSSA docs were migrated to MicrosoftDocs but not synced back to PSSA. Should I revert this change and sync the `com` change back to PSSA?

Comment (michaeltlombardi) [reply reference/ps-modules/PSScriptAnalyzer/Invoke-ScriptAnalyzer.md:517]: @bergmeister it looks like the switch to `com` was a mistake because that link is definitely broken - it should be `microsoft.powershell.core` which resolves correctly.

File: reference/docs-conceptual/PSScriptAnalyzer/Rules/AvoidSemicolonsAsLineTerminators.md [hunk 2]
```diff
@@ -14,8 +13,7 @@ title: AvoidSemicolonsAsLineTerminators
 
 Lines should not end with a semicolon.
 
-> [!NOTE]
-> This rule is not enabled by default. The user needs to enable it through settings.
+**Note**: This rule is not enabled by default. The user needs to enable it through settings.
```
Comment (sdwheeler) [inline reference/docs-conceptual/PSScriptAnalyzer/Rules/AvoidSemicolonsAsLineTerminators.md:16]:
```suggestion
> [!NOTE]
> This rule is not enabled by default. The user needs to enable it through settings.
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]

File: reference/ps-modules/PSScriptAnalyzer/Invoke-ScriptAnalyzer.md (hunk 1 above)
Comment (bergmeister) [reply reference/ps-modules/PSScriptAnalyzer/Invoke-ScriptAnalyzer.md:517]: Yes, you're right, thanks.

**Review by sdwheeler — APPROVED** [review]

**Comment by sdwheeler** [comment]
@bergmeister Thanks for submitting the PR. I have marked it as `hold-for-pr` awaiting your release.

**Comment by bergmeister** [comment]
Thanks @sdwheeler . Can you sync the change in your last [commit](https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/pull/99/commits/00ed70120d999f268d31d7bbe8f7bd848f57c03b)  back to PSSA please: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/docs/Rules/AvoidSemicolonsAsLineTerminators.md

**Comment by sdwheeler** [comment]
> Thanks @sdwheeler . Can you sync the change in your last [commit](https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/pull/99/commits/00ed70120d999f268d31d7bbe8f7bd848f57c03b) back to PSSA please: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/docs/Rules/AvoidSemicolonsAsLineTerminators.md

Yes.

**Comment by bergmeister** [comment]
@sdwheeler PSSA has been released now: https://github.com/PowerShell/PSScriptAnalyzer/releases/tag/1.21.0

**Review by michaeltlombardi — APPROVED** [review]
